### PR TITLE
Pe timeunits

### PR DESCRIPTION
**Pull request by Paulie14**
solving issue #1321

### CODE REVIEW

File: src/tools/time_governor.cc [hunk 1]
```diff
@@ -48,8 +49,9 @@ const Tuple & TimeGovernor::get_input_time_type(double lower_bound, double upper
     return Tuple("TimeValue", "A time with optional unit specification.")
         .declare_key("time", Double(lower_bound, upper_bound), Default::obligatory(),
                                     "The time value." )
-		.declare_key("unit", String(), Default::read_time("Common time unit of the equation's Time Governor."),
-				"The time unit. Possible values: 's' seconds, 'min' minutes, 'h' hours, 'd' days, 'y' years.")
+		.declare_key("unit", UnitConverter::get_input_type(),
+                Default::read_time("The time unit."),
```
Comment (jbrezmorf) [inline src/tools/time_governor.cc:53]: Keep previous default value description. Related is `TimeGovernor.common_time_unit` that has to be changed as well.

File: src/tools/time_governor.cc [hunk 2]
```diff
@@ -48,8 +49,9 @@ const Tuple & TimeGovernor::get_input_time_type(double lower_bound, double upper
...
+				"User can benefit from the Unit Convertor funcionality, e.g. creating own time units.")
```
Comment (jbrezmorf) [inline src/tools/time_governor.cc:54]: Have to check all places accessing "time" and "unit" and call `UnitConverter::convert("s")`.

Comment (jbrezmorf) [reply src/tools/time_governor.cc:54]: Seems that TimeUnitConversion is the right place.

**Review by jbrezmorf — COMMENTED** [review]
see comments
+ make an integration test with common_time_unit in 02_gneric_input

**Comment by Paulie14** [comment]
`common_time_unit` is already tested in 02/11, 02/12, 02/17 - I suppose that this is satisfying

**Pull request merged**